ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -2154,6 +2155,115 @@
             "metadata": {},
             "execution_count": 0,
             "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Variable for Recording the Logs"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "The following variables are defined in order to keep the file information and keep trace of the logs, each time we access the DB."
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Variable that Tracks the DB Activity"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public record KeyedActivity : KeyedRecord",
```

Comment:
These are technical types, and we should not add to the Data Structure, as this was originally thought to have only types with  meaning (related to the the IFRS content).
I would say those belong to something like infrastructure and we should have a notebook structure there.

Reply:
I moved this now to a dedicated notebook in the Utils.